This is the outcome of the unit test:
![Text message](https://github.com/SS-CC-YY/in_class_5/blob/master/unittest_outcome.png)

It seems all three test are success.

This is the outcome of the pytest:

![Text message](https://github.com/SS-CC-YY/in_class_5/blob/master/pytest_outcome.png)


This is the outcome of the coverage report:

![Text message](https://github.com/SS-CC-YY/in_class_5/blob/master/coverage_rep.png)
For the leap_year.py, I quite have no idea how I can cover the line 11-12, 24-26
For line 11-12 it is used to tell the user the input is invalid and let user to input another number, so it does not have any return so I have no idea how to test it.

Fpr 24-26 it is the main function, which is used to call the check function so I think it do not need test


The outcome of coverage html:
![Text message](https://github.com/SS-CC-YY/in_class_5/blob/master/html_result_1.png)
![Text message](https://github.com/SS-CC-YY/in_class_5/blob/master/html_result_2.png)
![Text message](https://github.com/SS-CC-YY/in_class_5/blob/master/pytest_cov.png)

The result is the same as the coverage report, and as what I said above, I quite have no idea how to test line 11-12

The coutcome of pytest -cove:
![Text message](https://github.com/SS-CC-YY/in_class_5/blob/master/html_result_1.png)
![Text message](https://github.com/SS-CC-YY/in_class_5/blob/master/html_result_2.png)
![Text message](https://github.com/SS-CC-YY/in_class_5/blob/master/pytest_cov.png)
